alleycats-core/src/main/scala/alleycats/std/iterable.scala

```diff
@@ -0,0 +1,38 @@
+package alleycats
+package std
+
+import cats.{Eval, Foldable}
+import export._
+
+@reexports(IterableInstances)
+object iterable extends LegacyIterableInstances
+
+@exports
+object IterableInstances {
+  @export(Orphan)
+  implicit val exportIterableFoldable: Foldable[Iterable] =
+    new Foldable[Iterable] {
```

Comment:
I don't actually know why this shouldn't be in cats-core. What is the problem exactly? `Foldable` is lawless to begin with.

Reply:
AFAIK `Iterable`'s only method is `iterator` which isn't exactly referentially transparent, no?

Reply:
But that is the only method you need to implement Foldable. And I don’t see why creating a new Iterator is not referentially transparent. It certainly is for all instances of Iterable is the standard library AFAIK.

Reply:
Doesn’t Stream have a Foldable instance in cats already? What prevents it from being Foldable?

Reply:
you are right, `Stream` is `Foldable`. Yeah, I don't see why we couldn't move `Foldable` of`Iterable` into core.